java/src/jmri/jmrix/nce/NceBinaryCommand.java

```diff
@@ -134,6 +135,7 @@ normal binary number (NOT in DCC format).
 
     // NCE Command 0xA2 sends speed or function packets to a locomotive
     // 0xA2 sub commands speed and functions
+    public static final byte LOCO_CMD_NOP = 0x00;          // No Operation (Not documented by NCE)
```

Comment:
This is now documented as 'select loco' command  (OP Code 0x00, data 0x00) in the latest (unreleased) Power Pro release notes.